test/acceptance/features/bindAppToService.feature

```diff
@@ -570,6 +570,8 @@ Feature: Bind an application to a service
             kind: Backend
             metadata:
                 name: demo-backserv-cr-3
+                annotations:
+                    service.binding/name: path={.metadata.name}
```

Comment:
how is this change related to PR change?

Reply:
The PR makes it necessary to have some binding data in order to finish binding successfully and to make it 'Ready'. Those 3 scenarios did not have anything bindable and so with introducing this PR it failed at first SB being checked for Readiness. So I have to introduce some binding data for those scenarios to work.